docs/knowledge_base/ext_knowledge.md

```
@@ -297,13 +297,13 @@ datus-cli --datasource california_schools
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)